### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by NightTsarina**
* 2926a50 - Update README with cmdline changes
* 4d4d33d - Improve README
  * Clarify the type of devices supported.
  * Document how to run without root.
  * Update Docker instructions to use current code.

### CODE REVIEW

**Comment by NightTsarina** [comment]
@dswarbrick After a couple of years, I finally sat down to fix some  bugs in this exporter, so hopefully I can update the Debian package soon too. And since you worked on this too in the past, I am sending you some pull requests to have a second pair of eyes. Hope that is OK, no worries if you can't!

File: README.md
Comment (dswarbrick) [inline README.md]: Unnecessary caps.
```suggestion
HomePlug devices on the local layer 2 network segment.  This will NOT find
```

File: README.md
Comment (dswarbrick) [inline README.md]: Unnecessary caps.
```suggestion
on the same layer 2 network segment as at least one HomePlug device.
```

File: README.md
Comment (dswarbrick) [inline README.md]: Should "powerline" be written as "Power Line" as it is elsewhere in the document?

**Review by dswarbrick — CHANGES_REQUESTED** [review]
Just a couple of minor nits, otherwise LGTM.

File: README.md
Comment (NightTsarina) [reply README.md]: Good point, I think I will just use PLC and adjust a few other inconsistencies this made me reallise

Comment (dswarbrick) [reply README.md]: Hmm, PLC says "programmable logic controller" to my brain, as I've worked with such devices in the past. I'd be wary of abbreviating it as such.

edit: I see that Wikipedia abbreviates "power-line communication" as PLC, so... :man_shrugging:

Comment (NightTsarina) [reply README.md]: Yeah, that can be a bit confusing, but it seems to be the term used. Also, it helps differentiate when talking about "power lines" in general

File: README.md
Comment (dswarbrick) [inline README.md]: Last nit (hopefully). I should really review PRs when I'm more awake :sleeping: 
```suggestion
Currently it does not support other chipsets as it uses Atheros-specific
```

**Review by dswarbrick — APPROVED** [review]
Preemptively approving this, assuming that you will fix that last typo.

File: README.md
Comment (NightTsarina) [reply README.md]: No worries, thanks for the attention to detail!

**Pull request merged**
